cartographer/mapping_3d/sparse_pose_graph/optimization_problem.cc

```diff
@@ -32,6 +32,7 @@
 #include "cartographer/mapping_3d/acceleration_cost_function.h"
 #include "cartographer/mapping_3d/ceres_pose.h"
 #include "cartographer/mapping_3d/rotation_cost_function.h"
+#include "cartographer/mapping_3d/imu_integration.h"
```

Comment:
Please sort alphabetically. This belongs one line up. Is `clang-format` not doing this for you?